public/static/app/perfdiscovery/PerfDiscoveryService.js

```diff
@@ -159,19 +159,22 @@ mciModule.factory('PerfDiscoveryService', function($q, ApiV1, ApiTaskdata) {
       .map(function(item, id) {
         var baseline = results.baseline[id]
         var appendix = {
-          ratio: ratio(item, baseline.speed),
+          ratio: ratio(item.speed, baseline.speed),
           baseSpeed: baseline.speed,
           trendData: _.chain(results.history)
+            .take(100)
```

Comment:
Is the ```take(100)```  change permanent?

Reply:
If you mean 'destructive', no, it isn't. Moreover, I think all `_.underscore` functions are non-destructive.